src/Action/SetObjectFieldValueAction.php

```diff
@@ -96,6 +96,11 @@ public function __invoke(Request $request): JsonResponse
         }
 
         if ('list' === $context) {
+            /**
+             * NEXT_MAJOR: remove annotation. Return type of interface will be correct.
+             *
+             * @var FieldDescriptionInterface|null
```

Comment:
This is the same, please leave it in the interface. (On master we will change it to not return null. (And probably will need to adapt some of this code.